examples/stable_diffusion/stable_diffusion_docker.yaml

```diff
@@ -1,23 +1,24 @@
 #SkyPilot YAML to run stable diffusion web tool on 1 V100 GPU.
 
 resources:
+  cloud: gcp
   accelerators: V100:1
 
 file_mounts:
   /stable_diffusion: .
 
 setup: |
-  sudo rm -r stable-diffusion-webui-docker
-  git clone https://github.com/AbdBarho/stable-diffusion-webui-docker.git
   sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
   sudo chmod +x /usr/local/bin/docker-compose
+  cd stable-diffusion-webui-docker  sudo rm -r stable-diffusion-webui-docker
+  git clone https://github.com/AbdBarho/stable-diffusion-webui-docker.git
   cd stable-diffusion-webui-docker
   git reset --hard 0d8b7d4ac8f9ba99e041ca332547eab9d65e6360
   wget https://www.googleapis.com/storage/v1/b/aai-blog-files/o/sd-v1-4.ckpt?alt=media -P models
   mv models/'sd-v1-4.ckpt?alt=media' models/model.ckpt
-  rm build/Dockerfile
-  cp /stable_diffusion/Dockerfile build/
-  docker-compose build
+  docker pull berkeleyskypilot/sbtest
```

Comment:
This is just what I happened to name (StaBle Diffusion Test) it when pushing the image to DockerHub. I can modify the image name locally, but it will require pushing it again to DockerHub I believe.

Reply:
Are there any drawbacks to push it again to DockerHub?

Reply:
None, just pushed again to DockerHub and realized the push only takes a few seconds when the underlying image is not modified.